src/store/normalize-data.js

```diff
@@ -91,7 +81,7 @@ const formatData = data => {
       data.tags.forEach(addTag(state));
     }
     if (data.layers) {
-      data.layers.forEach(addLayer(state));
+      state.layer.ids = data.layers;
```

Comment:
There are two options here:

1. Keep the layer IDs array and the layer names object intact, and populate them both with the layer name. This keeps some redundant code around as they're duplicates, but it futureproofs it if we need a separate layer ID in the future for whatever reason.

2. The alternative is to remove them both and replace the state layer property (`layer: { ids: [], name: {} }`) with `layer: []`.

Instead, in this PR you seem to be going for both options incompletely at the same time. I think it's better to commit to one of them properly. Can I please request that you either update the layer state shape everywhere it's touched (i.e. in `initial-state.js`, and in the layer selector where it creates both an id and a name field, and anywhere else I've forgotten), OR you restore the `addLayer` function in the data normalizer, and modify it so that it sets layer id and name to be equal to the layer name.

Reply:
I will go with option 1. Thanks.